os/drivers/audio/syu645b.c

```diff
@@ -59,6 +57,11 @@
 #define CONFIG_SYU645B_NUM_BUFFERS       4
 #endif
 
+/* Converting buffer size from bytes to bits */
+#define SYU645B_I2S_TRANSFER_TIME_MS     (CONFIG_SYU645B_BUFFER_SIZE * CONFIG_SYU645B_NUM_BUFFERS * 8 * 1000) / (SYU645B_SAMPLE_FREQUENCY_IN_HZ * SYU645B_CHANNELS * SYU645B_BPSAMP)
```

Comment:
Ideally we should be able to change the sampling rate and channels. So, we should not use hardcoded values. I think instead we can use,

```
                priv->samprate = caps->ac_controls.hw[0];
                priv->nchannels = caps->ac_channels;
                priv->bpsamp = caps->ac_controls.b[2];
```